liana/plotting/_dotplot.py

```diff
@@ -249,7 +260,8 @@ def _prep_liana_res(adata=None,
                     target_labels=None,
                     colour=None,
                     size=None,
-                    uns_key='liana_res'):
+                    uns_key='liana_res', 
+                    met = False):
```

Comment:
No flags should be visible to the user, this should be essentially the same plot.

Reply:
but what to do, if users want to run both ? then there should be different names right ? or do we append the results table then ? could be complicated if different scores are used